### ISSUE TITLE
Option to replace or remove the "no event"

### ISSUE THREAD

**Issue by ArjenSwBase** [antoniandre/vue-cal-v4#42]
Hello,

Could we have an extra option that could either replace the "No event" text to something else like "Nothing planned" or if left empty completely removed? default of course still "No event".

Thanks!

**Comment by antoniandre** [comment]
Hi @ArjenSwBase,

Fair enough, I will think about a nice way to allow overriding it.

**Comment by antoniandre** [comment]
Hi @ArjenSwBase,

This is now possible from version 1.27.0.
Please refer to https://antoniandre.github.io/vue-cal/#ex--custom-title-and-no-event-text

**Comment by ArjenSwBase** [comment]
Hey @antoniandre 

Thanks for the quick development of this calendar!
I have an issue with the slot for no-event though it appears that if there are events present somewhere else in the view it will throw an error.

```
[Vue warn]: Duplicate presence of slot "no-event" found in the same render tree - this will likely cause render errors.

found in

---> <VueCal> at index.vue
```

**Comment by antoniandre** [comment]
Hi @ArjenSwBase, I have tried it with events and can't reproduce: https://codepen.io/antoniandre/pen/rPzWOJ.

Please fork this pen and place your code in to recreate the issue if it is still here.

**Comment by ArjenSwBase** [comment]
Hey @antoniandre 

I've tried to reproduce the bug in that codepen but I couldn't manage, even though in my own environment it consistently happend, even in clean new calendars.

So I decided to check what was different between your environment and mine; decided to update my vue to the newest 2.5.* version and it is now fixed. So I dove into the change log of vue to pinpoint what has fixed it and I think its this one: [https://github.com/vuejs/vue/releases/tag/v2.5.18-beta.0](https://github.com/vuejs/vue/releases/tag/v2.5.18-beta.0) 

This can be closed again! Thanks!
